### DISCUSSION TITLE
Polygon to line segments

### DISCUSSION THREAD

**Discussion by hdahmani1** [NetTopologySuite/NetTopologySuite#659] [Q&A] [answered]
Hi, 
Is there a build function to get the minimum distance between two point from a list of points ?
Thank you

**Comment by DGuidi** [comment, marked as answer]
you can easily build a LineString from the list of points with [`var ls = GeometryFactory.Default.CreateLineString`](https://nettopologysuite.github.io/NetTopologySuite/api/NetTopologySuite.Geometries.GeometryFactory.html#NetTopologySuite_Geometries_GeometryFactory_CreateLineString_NetTopologySuite_Geometries_Coordinate___), then use the [`ls.Distance`](https://nettopologysuite.github.io/NetTopologySuite/api/NetTopologySuite.Geometries.Geometry.html#NetTopologySuite_Geometries_Geometry_Distance_NetTopologySuite_Geometries_Geometry_) method.
